AIMSICD/src/main/java/com/secupwn/aimsicd/ui/activities/MainActivity.java

```diff
@@ -255,6 +264,10 @@ void selectDrawerItem(int position) {
 
         } else if (selectedItem.getId() == DrawerMenu.ID.APPLICATION.DOWNLOAD_LOCAL_BTS_DATA) {
             downloadBtsDataIfApiKeyAvailable();
+        } else if (selectedItem.getId() == DrawerMenu.ID.APPLICATION.IMPORT_CELL_TOWERS_DATA) {
+            Intent pickFileIntent = new Intent(Intent.ACTION_GET_CONTENT);
+            pickFileIntent.setType("*/*");
```

Comment:
I think the category `CATEGORY_OPENABLE` should be added here, so you can only select real files